Client/src/Components/Profile.jsx

```diff
@@ -0,0 +1,12 @@
+
+
+
+function Profile() {
+  return (
+    <div>
+      <h1>THIS is Profile page</h1>
```

Comment:
Add meaningful content or components inside the Profile page to utilize the space effectively, as it currently only contains a placeholder text. [medium]